### PR TITLE
Add Robert support

### PR DESCRIPTION
**Pull request by Hexxeh**

### CODE REVIEW

**Comment by Katharine** [comment]
0.0.22 doesn't exist, so there's no reason to jump to 0.0.23.

File: libpebble2/util/bundle.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ class PebbleBundle(object):
         'basalt': ('basalt/', ''),
         'chalk': ('chalk/',),
         'diorite': ('diorite/', 'aplite/', ''),
+        'emery': ('emery/', 'basalt/', 'aplite/', ''),
```
Comment (Katharine) [inline libpebble2/util/bundle.py]: `aplite/` isn't valid, since we don't support backwards compatibility with SDK 3 aplite apps.

**Comment by Hexxeh** [comment]
Previous PR added 0.0.22, but if it isn't on PyPi yet, I can keep this at 0.0.22 too.

**Comment by Katharine** [comment]
It's not yet.

File: libpebble2/util/hardware.py [hunk 2]
```diff
@@ -46,6 +48,7 @@ class PebbleHardware(object):
         SNOWY_BB2: 'basalt',
         SPALDING_BB2: 'chalk',
         SILK_BB: 'diorite',
+        ROBERT_BB: 'emery',
```
Comment (Katharine) [inline libpebble2/util/hardware.py:52]: If you've added `ROBERT_EVT`, you should probably add it to this mapping too.

**Pull request merged**